mailbox/plugin/deleted-messages-vault/src/test/java/org/apache/james/vault/DeletedMessageConverterTest.java

```diff
@@ -95,7 +96,9 @@ private MailboxMessage buildMessage(MessageBuilder messageBuilder, Collection<Me
         return SimpleMailboxMessage.fromWithoutAttachments(mailboxMessage)
             .mailboxId(mailboxMessage.getMailboxId())
             .internalDate(Date.from(DELIVERY_DATE.toInstant()))
-            .addAttachments(attachments).build();
+            .addAttachments(attachments)
+            .bodyStartOctet((int) mailboxMessage.getFullContentOctets() - CONTENT.length)
```

Comment:
Can't we split CONTENT into HEADER_CONTENT and BODY_CONTENT? then CONTENT would be the simple concat of it and we will avoid conplex computations like this...